packages/SymbioticController/composer.json

```diff
@@ -4,7 +4,8 @@
         "php": "^7.1",
         "nette/di": "^2.4",
         "nette/application": "^2.4",
-        "odbav-to/presenter-route": "^0.1"
+        "odbav-to/presenter-route": "^0.1",
```

Comment:
I've opened new PR, so it's easy to see: https://github.com/Symplify/Symplify/pull/139